src/init/init.ts

```diff
@@ -193,6 +198,41 @@ function createSelectPrompt(env: YeomanEnvironment) {
   };
 }
 
+/**
+ * Checks and returns proxy settings from gitconfig. This function
+ * assumes that gitconfig will either be located locally (project's config file)
+ * or globally (user home)
+ */
+function getProxyConfig() {
+  let pathToConfig = './.git/config';
```

Comment:
Instead of using `let` and reassigning variables within a function scope, we prefer to create new, clean variables with `const`. Can you rename to `pathToLocalConfig`, `localMeta`, `userMeta`, etc. (fwiw, the first `pathToConfig` value is only used once and doesn't need its own variable)

Reply:
Also, use `path.join()` when joining paths for windows support.